### PR TITLE
Auto rest

### PR DESCRIPTION
**Pull request by vrmurthy01**
Updated version of Nuget Microsoft.Rest.ClientRuntime.Azure.2.1.0

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@vrmurthy01__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (vrmurthy). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by vrmurthy01** [comment]
Folks,

Can this PR be prioritized? We need to link Powershell with the nuget generated from here and we are delayed on that.

Thanks,
Venkata

File: src/ResourceManagement/Intune/Intune/packages.config [hunk 1]
```diff
@@ -1,10 +1,11 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Rest.ClientRuntime" version="1.4.1" targetFramework="net45" />
-  <package id="Microsoft.Rest.ClientRuntime.Azure" version="2.0.0" targetFramework="net45" />
-  <package id="Microsoft.Rest.ClientRuntime.Azure.Authentication" version="0.10.0" targetFramework="net45" />
   <package id="Microsoft.Bcl" version="1.1.9" targetFramework="net45" />
   <package id="Microsoft.Bcl.Async" version="1.0.168" targetFramework="net45" />
   <package id="Microsoft.Bcl.Build" version="1.0.14" targetFramework="net45" />
+  <package id="Microsoft.IdentityModel.Clients.ActiveDirectory" version="2.18.206251556" targetFramework="net45" />
```
Comment (markcowl) [inline src/ResourceManagement/Intune/Intune/packages.config]: Your sdk assembly should not depend on the activedirectory or authentication packages

File: src/ResourceManagement/Intune/Intune/Intune.csproj [hunk 2]
```diff
@@ -56,14 +56,21 @@
...
-        </Reference>
       </ItemGroup>
     </When>
   </Choose>
   <ItemGroup>
+    <Reference Include="Microsoft.IdentityModel.Clients.ActiveDirectory">
+      <HintPath>..\..\..\..\packages\Microsoft.IdentityModel.Clients.ActiveDirectory.2.18.206251556\lib\net45\Microsoft.IdentityModel.Clients.ActiveDirectory.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
+    <Reference Include="Microsoft.IdentityModel.Clients.ActiveDirectory.WindowsForms">
```
Comment (markcowl) [inline src/ResourceManagement/Intune/Intune/Intune.csproj]: You should not have this dependency

File: src/ResourceManagement/Intune/Intune/Intune.csproj [hunk 3]
```diff
@@ -56,14 +56,21 @@
...
         </Reference>
-        <Reference Include="Microsoft.Rest.ClientRuntime.Azure">
-          <HintPath>$(LibraryNugetPackageFolder)\Microsoft.Rest.ClientRuntime.Azure.2.0.0\lib\portable-net45+win8+wpa81\Microsoft.Rest.ClientRuntime.Azure.dll</HintPath>
-          <Private>True</Private>
-        </Reference>
       </ItemGroup>
     </When>
   </Choose>
   <ItemGroup>
+    <Reference Include="Microsoft.IdentityModel.Clients.ActiveDirectory">
```
Comment (markcowl) [inline src/ResourceManagement/Intune/Intune/Intune.csproj]: You should not have this dependency

**Comment by markcowl** [comment]
Please update your nuspec file for the new runtime.azure dependency as well

File: src/ResourceManagement/Intune/Intune/Intune.csproj (hunk 3 above)
Comment (vrmurthy01) [inline src/ResourceManagement/Intune/Intune/Intune.csproj]: Removed

File: src/ResourceManagement/Intune/Intune/packages.config (hunk 1 above)
Comment (vrmurthy01) [inline src/ResourceManagement/Intune/Intune/packages.config]: taken care

**Comment by vrmurthy01** [comment]
It already has the dependency info:
      <group targetFramework="portable-net45+win8+wpa81">
        <dependency id="Microsoft.Rest.ClientRuntime.Azure" version="[2.0,3.0)" />
      </group>

**Pull request merged**